charts/synthetics-minion/templates/statefulset.yaml

```diff
@@ -93,6 +93,13 @@ spec:
               value: {{ include "synthetics-minion.user-defined-variables-path" . | quote }}
           volumeMounts:
           {{- include "synthetics-minion.volumeMounts" . | nindent 12 }}
+          resources:
+            requests:
+              cpu: "100m"
+              memory: "100Mi"
+            limits:
+              cpu: "200m"
+              memory: "200Mi"
```

Comment:
These resources need to be here if a namespace quota is in use.

Reply:
I entered what should be reasonable values, but feel free to adjust up if needed.